src/app/components/BibPage/BibPage.jsx

```diff
@@ -155,10 +153,27 @@ class BibPage extends React.Component {
     return detailFields;
   }
 
+  getResultsLink(searchURL) {
+    if (!searchURL) {
+      return null;
+    }
+
+    return (
+      <Link
+        title={`Go back to search results for ${this.props.searchKeywords}`}
+        className="nypl-back-link"
+        to={`/search?${searchURL}`}
+      >
+        <LeftArrowIcon />
+        Back to Search Results
+      </Link>
+    );
+  }
+
   render() {
     const createAPIQuery = basicQuery(this.props);
     const record = this.props.bib ? this.props.bib : this.props.item;
-    const bibId = record['@id'].substring(4);
+    const bibId = record && record['@id'] ? record['@id'].substring(4) : '';
```

Comment:
We can expect the API to give us a string, it may be too much checking to see if it's a string.